crates/bevy_ecs/src/system/commands/mod.rs

```diff
@@ -386,6 +386,48 @@ impl<'w, 's> Commands<'w, 's> {
     pub fn add<C: Command>(&mut self, command: C) {
         self.queue.push(command);
     }
+
+    /// Adds a [`BoxedCommand`] directly to the command list.
```

Comment:
Could the docstr mention why one would use a boxed command rather than the usual?

Reply:
Mentioned and gave a good example.

Reply:
Thank you -- thorough and really helpful :tada: